.github/workflows/ci.yaml

```diff
@@ -93,6 +93,9 @@ jobs:
   drenv-test:
     name: drenv tests
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Cool! but I'm not sure we want to waste time on very old python. I think supporting last 2 or 3 releases is good enough, so maybe test only on 3.9, 3.10, 3.11?